lib/AACStream/aacs.js

```diff
@@ -0,0 +1,123 @@
+/**
```

Comment:
Following the naming conventions in the rest of the project, this file should be `aac.js` and be located in a directory called `aac`. You could also just make this file be the index.js in `aac`, I would be ok with that and it'll reduce the file count by one.